### ISSUE TITLE
Add perftype to filenames to prevent unfulfilled dependency errors

### ISSUE THREAD

**Issue by guidopetri** [guidopetri/chess-pipeline#61]
For some time now, I've been getting spurious "unfulfilled dependency at runtime" errors, and I think I've finally figured it out: A task starts because its `self.complete` function returns `False`, but then *during* the runtime of that task, another task starts that writes to the exact same file, and thus makes the first task's `self.complete` function return `True`.

I think the above is the case, but I need to investigate closer. If so, the culprit is not adding perftype to the filenames that all tasks write to - I run both blitz and bullet for a handful of users, and if one of them contains more games than the other, it'll "run over" to the next task's allotted time, thus creating the above error.

**Comment by guidopetri** [comment]
Obviously, if the entire pipeline runs at the same time that another instance of the pipeline is running, then the temp files will get purged... which breaks the longer-running pipeline.

**Comment by guidopetri** [comment]
Hopefully closed with 7f84fb3 . This commit makes a `CopyWrapper` only remove the targets that are involved in its completion, not completely clean the temp files.

**Comment by guidopetri** [comment]
(Testing this tonight by playing some bullet games!)

**Comment by guidopetri** [comment]
I neglected the fact that `PostgresTarget`s won't have a `.path` attribute. Fixed this in f4993ea .

I also neglected the fact that `CopyWrapper`'s inputs are all `PostgresTarget`s, and I should actually be looking at the jobs list itself and iterating over the functions there. Fixed in 34fce4d .

Hopefully this is the actual end of this issue's saga. The lesson: I need a better testing, probably through a staging/test environment.
